backend/dal/Services/ResearchFileService.cs

```diff
@@ -85,5 +84,37 @@ public Paged<PimsResearchFile> GetPage(ResearchFilter filter)
             _user.ThrowIfNotAuthorized(Permissions.ResearchFileView);
             return _researchFileRepository.GetPage(filter);
         }
+
+        public PimsResearchFile UpdateProperty(long researchFileId, long researchFilePropertyId, long researchFileVersion, PimsPropertyResearchFile propertyResearchFile)
+        {
+            _logger.LogInformation("Updating property research file...");
+            _user.ThrowIfNotAuthorized(Permissions.ResearchFileEdit);
+            ValidateVersion(researchFileId, researchFileVersion);
+            return _researchFileRepository.UpdateProperty(researchFileId, propertyResearchFile);
+        }
+
+        private void PopulateResearchFile(PimsProperty property)
+        {
+            property.PropertyClassificationTypeCode = "CORESTRAT"; // Todo: should be "UNKNOWN"
```

Comment:
same comment as last time for visibility, are we tracking on these TODOs? Doug should be made aware of this one for example.

Reply:
I've updated it since the type exists in the db